internal-enrichment/virustotal/src/config.yml.sample

```diff
@@ -3,14 +3,35 @@ opencti:
   token: 'ChangeMe'
 
 connector:
-  id: 'ChangeMe'
+  id: 'VirusTotal'
   type: 'INTERNAL_ENRICHMENT'
   name: 'VirusTotal'
-  scope: 'StixFile,Artifact'
+  scope: 'StixFile,Artifact,IPv4-Addr,Domain-Name,Url'
```

Comment:
Do you maybe also want to add IPv6 addresses?

Reply:
> Do you maybe also want to add IPv6 addresses?

These changes do not include support for IPv6 as they are outside our use cases